src/common/parallel_for.cpp

```diff
@@ -111,20 +142,30 @@ void ParallelFor::par_for_mask(
     /* Pre-calculate the masks to simplify the increment function later. */
     bitCapInt masks[maskLen][2];
 
+    bool allLow = true;
```

Comment:
Rename to `onlyLow` maybe to avoid code reading hiccups to allow?

Reply:
Done.